subprojects/docs/src/docs/userguide/authoring-builds/tutorial_using_tasks.adoc

```diff
@@ -241,6 +241,8 @@ include::{snippetsPath}/tutorial/antLoadfileWithMethod/tests/antLoadfileWithMeth
 ----
 ====
 
+WARNING: Accessing top-level methods and variables is not yet compatible with the <<configuration_cache#config_cache:not_yet_implemented:accessing_top_level_at_execution,configuration cache>>.
```

Comment:
```suggestion
WARNING: Accessing top-level methods and variables is not compatible with the <<configuration_cache#config_cache:not_yet_implemented:accessing_top_level_at_execution,configuration cache>>.
```

Suggestion: remove "yet" since docs are evergreen, and we don't necessarily know when or if these will be compatible. Could be months, could be years -- better to document the feature as-is right now, and remove this note when we make this compatible.